src/test/testing-library/ForWhoSection.test.tsx

```
@@ -9,15 +9,26 @@ const forWhoLabel: string = t('for_who.aria_label');
-
     expect(getAllByLabelText(forWhoLabel)[0]).toBeInTheDocument();
   });
 
-  it('should have a proper alt tag', () => {
+  it('should have correct number of decorative images with empty alt', () => {
     const { getAllByAltText } = render(<ForWhoSection />);
-    const images: HTMLElement[] = getAllByAltText('Vector');
 
-    expect(images).toHaveLength(5);
-    expect(images[0]).toBeInTheDocument();
+    const decorativeImages = getAllByAltText('');
```

ESLint

[error] Expected decorativeImages to have a type annotation.

(@typescript-eslint/typedef)